source/includes/code-examples/change-streams/change-streams.cs

```diff
@@ -0,0 +1,100 @@
+// start-open-change-stream
+var database = client.GetDatabase("sample_restaurants");
+var collection = database.GetCollection<Restaurant>("restaurants");
+
+// Open a change stream and print the changes as they're received
```

Comment:
S: based on the code comment style guide [doc](https://docs.google.com/document/d/1Bnhh1OsGRUuXo_6yVUASGWiSX6sWuqrEFxPP6sLZDHM/edit#heading=h.xv6glj2lnzjh), I'd use the indicative tense rather than imperative
```suggestion
// Opens a change stream and prints the changes as they're received
```

Reply:
Applies to all